js/src/alert.js

```diff
@@ -49,7 +49,7 @@ class Alert {
     this._element = element
 
     if (this._element) {
-      Data.setData(element, DATA_KEY, this)
```

Comment:
BTW the `DATA_KEY` constants are only used in one place in each of the plugins except for popover.js and tooltip.js. The minifier already inlines those, so I guess we better keep them for consistency.